components/typeform/actions/list-responses/list-responses.mjs

```diff
@@ -81,13 +82,13 @@ export default {
     completed: {
       type: "boolean",
       label: "Completed",
-      description: "Limit responses only to those which were submitted. This parameter changes `since/until` filter, so if `completed=true`, it will filter by `submitted_at`, otherwise - `landed_at`.",
+      description: "Limit responses only to those which were submitted. This parameter changes `since`/`until` filter, so if `completed=true`, it will filter by `submitted_at`, otherwise - `landed_at`.",
```

Comment:
if I set `completed` to `false` and I don't manually set `sort` to `landed_at`, this will return a 400. are we able to handle that on the user's behalf?